packages/DOS/common.dj

```diff
@@ -61,15 +59,30 @@ else
   DS     = \$(NOTHING)
 endif
 
+ifeq ($(OS),Windows_NT)
+  #
+  # Windows hosted djgpp cross compiler. Get it from:
+  #   https://github.com/andrewwutw/build-djgpp/releases
+  #
+  DJ_PREFIX ?= c:/some-path/djgpp/bin/i586-pc-msdosdjgpp-
+  CC = $(DJ_PREFIX)gcc
+
+else
+  #
+  # The normal djgpp 'gcc' for MSDOS.
+  #
+  CC = gcc
+endif
+
 #
 # OpenSSL is available from www.openssl.org and builds okay
 # with djgpp/Watt-32. Set to 0 if you don't need https URLs
 # (reduces curl.exe with approx 700 kB)
 #
-USE_SSL = 0
+USE_OPENSSL = 0
 
 #
-# Use zlib for contents encoding
+# Use zlib for contents encoding. Needed for 'USE_OPENSSL=1' too.
```

Comment:
Since when is zlib needed for OpenSSL?

Reply:
It could be when `perl Configure enable-zlib` is used (what I used last time I built OpenSSL with djgpp).
No use of zlib.dll on MSDOS you know.

Reply:
> It could be when `perl Configure enable-zlib` is used (what I used last time I built OpenSSL with djgpp).

Then I think you could say something like # May be needed if using static OpenSSL built with zlib enabled.

Reply:
Okay. But how about the syntax `USE_ZLIB ?= 0`. So one could do a `make -f Makefile.dj USE_ZLIB=1 clean all`.

Reply:
> But how about the syntax `USE_ZLIB ?= 0`

They should probably all be like that since there is no configure script

Reply:
Done that now.